### PR TITLE
Add context and remove dead code

### PR DESCRIPTION
**Pull request by gfr10598**
This change is [Reviewable](https://reviewable.io/reviews/m-lab/etl/256)

### CODE REVIEW

**Comment by yachang** [comment]
Review status: 0 of 6 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[annotation/geo.go, line 79 at r2](https://reviewable.io:443/reviews/m-lab/etl/256#-KvXUYuylnroTiNjWFU3:-KvXUYuylnroTiNjWFU4:b-1gc7em) ([raw file](https://github.com/m-lab/etl/blob/adfbecea98268412a9166c7f72b973d17a1de5ed/annotation/geo.go#L79)):*
> ```Go
> }
> 
> // TODO(JosephMarques) See if there is a better way of determining
> ```

Remove Joseph from this TODO.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/256)*

File: annotation/geo.go [hunk 1]
```diff
@@ -42,6 +37,45 @@ func EnableAnnotation() {
...
+
+// The RequestData schema is the schema for the json that we will send
+// down the pipe to the annotation service.
+type RequestData struct {
+	IP        string    // Holds the IP from an incoming request
+	IPFormat  int       // Holds the ip format, 4 or 6
+	Timestamp time.Time // Holds the timestamp from an incoming request
+}
+
 // TODO(JosephMarques) See if there is a better way of determining
```
Comment (yachang) [inline annotation/geo.go]: Remove Joseph from this TODO

File: parser/util_test.go [hunk 2]
```diff
@@ -37,12 +37,10 @@ func TestValidateIP(t *testing.T) {
 	}
 }
 
+// To run benchmark...
+// go test -bench=. ./parser/...
```
Comment (yachang) [inline parser/util_test.go]: Thanks!

**Comment by gfr10598** [comment]
Review status: 0 of 6 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[annotation/geo.go, line 79 at r2](https://reviewable.io:443/reviews/m-lab/etl/256#-KvTDGxMp51k1yNE8M9h-r2-79:-KvXpm3sNBzsJPA8CiPI:b-896fix) ([raw file](https://github.com/m-lab/etl/blob/adfbecea98268412a9166c7f72b973d17a1de5ed/annotation/geo.go#L79)):*
<details><summary><i>Previously, yachang wrote…</i></summary><blockquote>

Remove Joseph from this TODO
</blockquote></details>

Done.

---

*[annotation/geo.go, line 79 at r2](https://reviewable.io:443/reviews/m-lab/etl/256#-KvXUYuylnroTiNjWFU3:-KvXpmTn9tEcoOL0VYT_:b-896fix) ([raw file](https://github.com/m-lab/etl/blob/adfbecea98268412a9166c7f72b973d17a1de5ed/annotation/geo.go#L79)):*
<details><summary><i>Previously, yachang wrote…</i></summary><blockquote>

Remove Joseph from this TODO.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/256)*

**Comment by gfr10598** [comment]
Review status: 0 of 6 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[parser/util_test.go, line 41 at r2](https://reviewable.io:443/reviews/m-lab/etl/256#-KvSBmJNH3m5Bz7m12FC-r2-41:-KvXpsPskFZUREI5hmtx:b-896fix) ([raw file](https://github.com/m-lab/etl/blob/adfbecea98268412a9166c7f72b973d17a1de5ed/parser/util_test.go#L41)):*
<details><summary><i>Previously, yachang wrote…</i></summary><blockquote>

Thanks!
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/256)*

**Comment by gfr10598** [comment]
PTAL?

---

Review status: 0 of 6 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/256#-:-KvXpuqfJ60q_jWV21kI:b-833305)*

**Comment by yachang** [comment]
LGTM

**Pull request merged**